specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json

```diff
@@ -1669,6 +1669,39 @@
           }
         }
       }
+    },
+    "/providers/Microsoft.AppPlatform/runtimeVersions": {
+      "get": {
+        "tags": [
+          "AppPlatform"
+        ],
+        "description": "You can directly pass \"{value[i].value}\" attribute (in the response) to all deployment CRUD operations.",
```

Comment:
Suppose should be "deployment creation operations" instead of "CRUD"? I think we don't support update runtime version today.